src/mumble/Plugins.cpp

```diff
@@ -229,27 +229,16 @@ Plugins::Plugins(QObject *p) : QObject(p) {
 		fPosition[i] = fFront[i] = fTop[i] = 0.0;
 	QMetaObject::connectSlotsByName(this);
 
-#ifdef QT_NO_DEBUG
-#	ifndef MUMBLE_PLUGIN_PATH
-	qsSystemPlugins =
-		QString::fromLatin1("%1/plugins").arg(MumbleApplication::instance()->applicationVersionRootPath());
-#		ifdef Q_OS_MAC
-	qsSystemPlugins = QString::fromLatin1("%1/../Plugins").arg(qApp->applicationDirPath());
-#		endif
-#	else
-	qsSystemPlugins = QLatin1String(MUMTEXT(MUMBLE_PLUGIN_PATH));
-#	endif
+	downloadPath = QLatin1String("%1/Plugins").arg(g.qdBasePath.absolutePath());
```

Comment:
```suggestion
	downloadPath = QString::fromLatin1("%1/Plugins").arg(g.qdBasePath.absolutePath());
```

should fix error on (Linux) CI

Reply:
Yep, that's the issue.